internal/consul/version.go

```diff
@@ -7,13 +7,16 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/hashicorp/terraform-provider-hcs/internal/helper"
-
+	cc_models "github.com/hashicorp/hcp-sdk-go/clients/cloud-consul-service/preview/2021-02-04/models"
```

Comment:
take it or leave it: it's not immediately clear to me what cc is from (consul-cloud?). To me cs (for consul-service) would be a little clearer.

Reply:
Taking it :) 👍